test/units/cli/test_data/collection_skeleton/docs/My Collection.md

```diff
@@ -0,0 +1 @@
+Welcome to my test collection doc for {{ namespace }}.
```

Comment:
It'd be interesting to add some Unicode symbols in the file name and see what happens :)

cc @abadger

Reply:
I originally tried that but it seems to not be a valid namespace or collection name according to our current rules https://github.com/ansible/ansible/blob/devel/lib/ansible/utils/collection_loader.py#L33

Reply:
So maybe write a negative test and test for failure?

Reply:
Have added it to https://github.com/ansible/ansible/pull/57106/files#diff-aaaff1b1dc993363be7a8c59decd5673R589 on the next commit.